packages/backend-core/src/security/roles.ts

```diff
@@ -366,6 +383,18 @@ export async function getAllRoles(appId?: string): Promise<RoleDoc[]> {
   }
 }
 
+async function shouldIncludePowerRole(db: Database) {
+  const app = await db.tryGet<App>(DocumentType.APP_METADATA)
+  const creationVersion = app?.creationVersion
```

Comment:
I'd be tempted to actually put this on the root `Document` type and call it `_creationVersion`, anticipating that we will likely introduce this idea onto other documents going forward.

Reply:
Yeah, then in `DatabaseImpl.post` we know when a doc is new because it has no ID, we could slap `_creationVersion` onto it there. Though it _is_ a wide-reaching change that likely has risk.

Reply:
I'm unsure about this - I don't think we need this on every document, since every DB will have an `app_metadata` (it won't load without this/is corrupt) - in theory noting it here is enough since its a static document that is always easy to retrieve.